systemtests/tests/py2plug-fd-local-fileset-basic/test-cleanup

```diff
@@ -0,0 +1 @@
+# this file is intentionally empty
```

Comment:
This is still required to be a well-formed executable (and thus bareos-check-sources wants a shebang).
```suggestion
#!/bin/sh
# this file is intentionally empty
```